Source/Raymarcher/Shaders/Private/WindowedRaymarchMaterials.usf

```diff
@@ -135,47 +135,106 @@ float4 PerformWindowedRaymarchOctree(Texture3D DataVolume, // Data Volume
 	// Copy the octree depth on mip level 0 for use when recalculating to correct texture coordinates.
 	float OctreeDepthConst = OctreeDepth;
 
-	// Get the size of the volume on requested mip. Used to recalculate coordinates to correct format.
-	OctreeVolume.GetDimensions(OctreeMip, OctreeWidth, OctreeHeight, OctreeDepth, OctreeMipCount);
+	// Get the bit number based on window center and window width.
+	uint WindowNumber = GetWindowingParamsBitNumber(WindowingParams.x, WindowingParams.y);
```

Comment:
This should be a uniform that's generated on the CPU from the window settings + transfer function opacity threshold